src/cascadia/TerminalApp/App.cpp

```diff
@@ -733,14 +734,21 @@ namespace winrt::TerminalApp::implementation
     void App::_UpdateTitle(std::shared_ptr<Tab> tab)
     {
         auto newTabTitle = tab->GetFocusedTitle();
+        const auto lastFocusedProfile = tab->GetFocusedProfile().value();
+        const auto* const matchingProfile = _settings->FindProfile(lastFocusedProfile);
+
+        auto tabTitle = matchingProfile->GetTabTitle();

```

Comment:
```suggestion
        const auto tabTitle = matchingProfile->GetTabTitle();
```

This can be const with the way you wrote it now.